File: kombarika/com/district/test/controller/DistrictController.java
package com.district.test.controller;




@
@RequestMapping(path = "district")
public class DistrictController
 {

	@Autowired
	private DistrictRepository repository;


	@PostMapping()
	public ResponseEntity<District> save(@RequestBody District district){
	 	return ResponseEntity.ok(repository.save(district));
	}
	@PutMapping()
	public ResponseEntity<District> update(@RequestBody District district){
	 	return ResponseEntity.ok(repository.save(district));
	}
	@DeleteMapping()
	public void delete(@RequestBody District district){
	 	return ResponseEntity.ok(repository.delete(district));
	}
	@GetMapping()
	public ResponseEntity<Iterable<District>> findAll(){
	 	return ResponseEntity.ok(repository.findAll());
	}





}

File: kombarika/com/district/test/controller/DistrictController.cs
namespace com.district.test.controller;


using System.Diagnostics;
using Microsoft.AspNetCore.Mvc;
using com.district.test.repository.DistrictContext;
using com.district.test.entity;
using ;


[ApiController]
[Route("api/[[district]]")]
public class DistrictController : Controller {

	private readonly DistrictDbContext _context;
	private readonly ILogger<DistrictController> _logger;


	[HttpPost]
	public ActionResult<District> save([FromBody] District district){
	 	_context.District.Add(district);
		_context.SaveChanges();
		return Ok();
	}
	[HttpPut]
	public ActionResult<District> update([FromBody] District district){
	 	var temp = district;
		_context.SaveChanges();
		return Ok();
	}
	[HttpDelete]
	public ActionResult<District> delete([FromBody] District district){
	 	_context.District.Remove(district);
		_context.SaveChanges();
	}
	[HttpGet]
	public ActionResult<IEnumerable<District>> findAll(){
	 	return Ok(_context.District.ToList());
	}

	public DistrictController(DistrictDbContext context) { _context = context; }



}
